### PR TITLE
Add channel link to message views

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: templates/channels/tags/channel_log_link.html [hunk 1]
```diff
@@ -1,5 +1,17 @@
-{% if logs_url %}
+{% load i18n %}
+
+{% if channel %}
   <div class="inline-block text-gray-400 linked ml-3">
+    <a href="{% url 'channels.channel_read' channel.uuid %}" onclick="goto(event, this)">
+      <temba-label icon="{{ channel.type.get_icon }}" secondary clickable class="my-1">
+        {% trans "Channel:" %}
```
Comment (rowanseymour) [inline templates/channels/tags/channel_log_link.html:7]: A PR like this needs a screenshot.. but this feels ugly.. maybe we can just have a tiny icon beside the channel log icon?

Comment (norkans7) [reply templates/channels/tags/channel_log_link.html:7]: @rowanseymour  You mean just the channel Icon and link to the channel?

**Comment by norkans7** [comment]
With the channel icon
Monosnap Flow Messages 2023-08-25 10-10-50

Monosnap Flow Messages 2023-08-25 10-11-46


Showing the name too(current changes in the PR)
Monosnap Flow Messages 2023-08-25 10-51-54

**Comment by norkans7** [comment]
I think having the name is useful so get the info quick without having to click on the link

**Comment by ericnewcomer** [comment]
Is this the icon for the channel, so it'll be a gray facebook icon or telegram icon, etc?

**Comment by norkans7** [comment]
Yes, this will be a gray icon close to the channel log icon

**Pull request closed without merging**